src/main/java/Command.java

```diff
@@ -0,0 +1,119 @@
+/**
+ * Represents a command handler to execute user inputs.
+ */
+public class Command {
+
+    private String typeOfCommand;
+    private String args;
+    boolean hasDescription = false;
+
+    public Command(String[] inputs) {
+        this.hasDescription = (inputs.length > 1);
+        this.typeOfCommand = inputs[0];
+
+        if(this.hasDescription) {
+            args = inputs[1];
+        }
+    }
+
+    /**
+     * Executes the user command based on user input.
+     *
+     * @param taskList class to store list of tasks at run time.
+     * @param ui Ui class for interaction with user.
+     * @param storage class for loading and saving of tasks to memory.
+     */
+    public void execute(TaskList taskList, Ui ui, Storage storage) {
```

Comment:
The method as many levels of nesting. Try to extract out some common operations as methods.
You can try to improve the SLAP here.